src/render/extract.rs

```diff
@@ -120,14 +120,14 @@ pub fn extract(
         Query<
             Entity,
             Or<(
-                Added<TilemapMeshType>,
-                Changed<TilemapMeshType>,
+                Added<TilemapType>,
+                Changed<TilemapType>,
                 Changed<GlobalTransform>,
                 Changed<TilemapTexture>,
                 Changed<TilemapTileSize>,
                 Changed<TilemapSpacing>,
                 Changed<TilemapGridSize>,
-                Changed<TilemapMeshType>,
+                Changed<TilemapType>,
```

Comment:
This is a duplicate of line 124, I'm actually surprised bevy doesn't complain! In any case can we remove one of these change filters?